src/popover/popover.scss

```diff
@@ -195,7 +195,7 @@
 // Responsive Popover
 // For < md it will show full-width anchored to the bottom
 
-@media (max-width: ($width-md - 0.02px)) {
+@media (max-width: calc(var(--width-md) - 0.02px)) {
```

Comment:
This needs to be a `calc` call since we are now using CSS variables and not the SASS compiler to add up these values.